recipe/bld.bat

```diff
@@ -0,0 +1,10 @@
+nmake -f psi\msvc.mak GSROOTDIR=%LIBRARY_PREFIX% WIN64= MSVC_VERSION=%VS_MAJOR% DEVSTUDIO=
+if errorlevel 1 exit 1
+
+move bin\*.dll %LIBRARY_BIN%
+cp bin\gswin64c.exe %LIBRARY_BIN%\gs.exe
+move bin\*.exe %LIBRARY_BIN%
+move bin\*.lib %LIBRARY_LIB%
+move bin\*.exp %LIBRARY_LIB%
+move Resource %LIBRARY_PREFIX%\share
+REM fonts
```

Comment:
Not sure where to put the `fonts` folder. Maybe `%LIBRARY_PREFIX%\share`?

Reply:
See https://www.ghostscript.com/doc/current/Use.htm#Finding_files. `GS_LIB_DEFAULT` is set by this line in `msvc.mak`:
```
GS_LIB_DEFAULT=$(GSROOTDIR)/Resource/Init;$(GSROOTDIR)/lib;$(GSROOTDIR)/Resource/Font;$(AROOTDIR)/fonts
```
So maybe one of those dirs (we don't set `AROOTDIR` ATM)? (I can't see how `GS_LIB_DEFAULT` gets passed into the build though, I must be missing something...)

Reply:
Maybe the `fonts` folder move it into `$(GSROOTDIR)/Resource/Font`? Does it make sense?

Reply:
Sounds good to me. I guess we'll find to whether this works when we try and use it...

Reply:
So `fonts` would go into `%LIBRARY_PREFIX%\Resource\Font`?

Reply:
Yep. Just fixed my last commit to reflect that.